titus-common/src/main/java/com/netflix/titus/common/runtime/internal/LoggingSystemLogService.java

```diff
@@ -0,0 +1,34 @@
+package com.netflix.titus.common.runtime.internal;
+
+import com.netflix.titus.common.runtime.SystemLogEvent;
+import com.netflix.titus.common.runtime.SystemLogService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class LoggingSystemLogService implements SystemLogService {
```

Comment:
to keep up with existing naming convention, should it be called DefaultSystemLogService  ?

Reply:
We use the term `Default` if there is no better qualifier. In this case it is specifically an implementation based on the logging library.